lock-api/src/main/java/com/palantir/lock/v2/LeaseContract.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.lock.v2;
+
+import java.time.Duration;
+
+public final class LeaseContract {
+    private LeaseContract() {}
+
+    public static final Duration LEASE_PERIOD = Duration.ofSeconds(10);
```

Comment:
this should really be in a server-only class because it's not really API

Reply:
it should probably also document when this can be changed (whenever!)

Reply:
I'd be tempted to also make this be a teeny bit smaller? Like, 8 seconds? So we _probably_ don't miss _two_ iterations of the lock refresher?